website/server/controllers/api-v4/faq.js

```diff
@@ -0,0 +1,83 @@
+import _ from 'lodash';
+import { query } from 'express-validator/check';
+import { langCodes } from '../../libs/i18n';
+import apiError from '../../libs/apiError';
+import common from '../../../common';
+
+const { replaceIdsWithTranslations, content } = common;
+const { faq } = content;
+
+const api = {};
+
+function _deleteProperties (obj, keysToDelete, platform) {
+  // if there is no description for specified platform, use 'web' description by default
+  if (obj[platform] === undefined) {
+    delete obj.ios;
+    delete obj.android;
+    return;
+  }
+
+  keysToDelete.forEach(key => delete obj[key]);
+}
+
+function _deleteOtherPlatformsAnswers (faqObject, platform) {
+  const faqCopy = _.cloneDeep(faqObject);
+  const keysToDelete = _.without(['web', 'ios', 'android'], platform);
+
+  _deleteProperties(faqCopy.stillNeedHelp, keysToDelete, platform);
+  faqCopy.questions.forEach(question => {
+    _deleteProperties(question, keysToDelete, platform);
+  });
+
+  return faqCopy;
+}
+
+/**
+ * @api {get} /api/v4/faq Get faq in json format
+ * @apiDescription Does not require authentication.
+ * @apiName FaqGet
+ * @apiGroup Content
+ *
+ * @apiParam (Query) {String="bg","cs","da","de",
+ *                   "en","en@pirate","en_GB",
+ *                    "es","es_419","fr","he","hu",
+ *                    "id","it","ja","nl","pl","pt","pt_BR",
+ *                    "ro","ru","sk","sr","sv",
+ *                    "uk","zh","zh_TW"} [language=en] Language code used for the items'
+ *                                                     strings. If the authenticated user makes
+ *                                                     the request, the content will return with
+ *                                                     the user's configured language.
+ *
+ *
+ * @apiSuccess {Object} data FAQ in a json format
+ */
+api.faq = {
+  method: 'GET',
+  url: '/faq',
+  middlewares: [
+    query('platform')
+      .optional()
+      .isIn(['web', 'android', 'ios']).withMessage(apiError('invalidPlatform')),
+  ],
+  async handler (req, res) {
+    const validationErrors = req.validationErrors();
+    if (validationErrors) throw validationErrors;
+
+    const proposedLang = req.query.language && req.query.language.toString();
+    const language = langCodes[proposedLang] !== null ? proposedLang : 'en';
```

Comment:
@paglias When I'm looking at it now it actually even worse, because langCodes is an array, not an object, so I think this way of checking whether value exists is kind of misleading (at least for me as a Java developer). I think I'll better change it to `langCodes.includes(proposedLang)`

Reply:
I mean it worked for sure, you can it by the completion of a test which specifically checks this behaviour (`'falls back to English if the desired language is not found'`), but it's better to change it anyways

Reply:
@paglias I think PR is ready, could you take a look pls here too?